### PR TITLE
upgrade jackson-databind

### PR DESCRIPTION
**Pull request by codeallthethingz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.5.1</version>
+            <version>[2.8.11.1,)</version>
```
Comment (brandonb927) [inline pom.xml]: is this supposed to be `)` or `]`?

Comment (codeallthethingz) [reply pom.xml]: )

Comment (codeallthethingz) [reply pom.xml]: The mysteries of pom versions.

**Comment by codeallthethingz** [comment]
Though it doesn't pass so I may have to bring this local to figure it out.  Shouldn't try and edit things through github. :)

**Pull request closed without merging**